pagopa/io-web-profile
# [IOPID-525] : created ko page in case of expired magic link

## Short description

New KO page and small graphic fix.

## List of changes proposed in this pull request

- Created KO Page in case the magic link has expired
- integrated a graphic fix for spinner during validation

## How to test

Run application
To check the new page go to http://localhost:3000/expiredMagicLink  
To check the style fix trigger validation and verify there is no blank page after the spinner stops
